packages/wonder-blocks-core/components/media-layout.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 import * as React from "react";
-import {View} from "@khanacademy/wonder-blocks-core";
+import View from "./view.js";
```

Comment:
🤷‍♂️ Do we have a way to lint against this? It feels like this is going to be an ongoing issue!